spec/automation/unit/method_validation/task_finished_spec.rb

```diff
@@ -27,19 +34,22 @@ def build_resolve_path
     FactoryGirl.create(:miq_provision_vmware, :provision_type => 'template',
                        :state => 'pending', :status => 'Ok',
                        :miq_request => miq_provision_request,
-                       :options => options, :userid => user.userid)
+                       :options => options, :userid => user.userid, :vm => vm)
   end
 
   it "miq_provision method succeeds" do
+    type = "automate_vm_provisioned"
```

Comment:
Create the Factory here using the `type` value as the `NotificationType` name.

`notification_type = FactoryGirl.create(:notification_type, :name =>"automate_vm_provisioned")`